smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HttpResponseBindingOutputGeneratable.kt

```diff
@@ -1,15 +0,0 @@
-package software.amazon.smithy.swift.codegen.integration.httpResponse
-
-import software.amazon.smithy.model.shapes.OperationShape
-import software.amazon.smithy.model.traits.TimestampFormatTrait
-import software.amazon.smithy.swift.codegen.integration.HttpBindingResolver
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-
-interface HttpResponseBindingOutputGeneratable {
```

Comment:
This interface is unneeded since there is now only one output generator implementation.